src/include/ipc/topology.h

```diff
@@ -47,6 +47,7 @@ enum sof_comp_type {
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
 	SOF_COMP_DCBLOCK,
 	SOF_COMP_SMART_AMP,		/**< smart amplifier component */
+	SOF_COMP_GENERIC_PROCESSOR,	/** generic processor component */
```

Comment:
@keyonjie dont we have this generic type upstream now ?

Reply:
@plbossart, if no new types then what SOF_COMP_* should this generic_processor be?